### PR TITLE
center rectangle shaded and setZoomByPrecision added

### PR DESCRIPTION
**Pull request by mridulnagpal**
#54

### CODE REVIEW

File: src/core/gridSystem.js [hunk 1]
```diff
@@ -11,8 +11,12 @@ module.exports = function gridSystem(options) {
                     {start: 2, end: 2, interval: 40},
                     {start: 3, end: 3, interval: 20},
```
Comment (jywarren) [inline src/core/gridSystem.js]: Let's remove any not a power of 10!

Comment (mridulnagpal) [inline src/core/gridSystem.js]: On it!

**Comment by mridulnagpal** [comment]
Done!

**Comment by mridulnagpal** [comment]
@jywarren I think this works please have a look

**Comment by mridulnagpal** [comment]
Okay So I have also added the center square shade option and it changes everytime you pan the map without altering any of the code in Leaflet,Graticule, please have a look at the demo itself updated there as well.
image

**Comment by mridulnagpal** [comment]
`setZoomByPrecision()` has been added and functional please have a look.

**Comment by jywarren** [comment]
Oh, awesome. This looks super super super!!!

I think maybe i found a small bug when i searched for `Somerville, MA` and then zoomed in a bit but did not drag at all:

screenshot 2017-07-24 at 7 35 13 pm

Pending that, good to merge!

**Comment by mridulnagpal** [comment]
Merging for now, will look into the bug.

**Comment by jywarren** [comment]
Great!

**Pull request merged**